test/uk/gov/hmrc/gform/gform/StructuredFormDataBuilderSpec.scala

```diff
@@ -30,7 +30,7 @@ import uk.gov.hmrc.gform.sharedmodel.formtemplate._
 import uk.gov.hmrc.gform.sharedmodel.structuredform._
 
 class StructuredFormDataBuilderSpec extends Spec {
-  implicit val l = LangADT.En
+  private implicit val l: LangADT.En.type = LangADT.En
```

Comment:
This is exactly what I was talking about yesterday, between nice distinction between types and values in Haskell. `LangADT.En` type we want to work with is `LangADT`, but type inference thinks something else 😞